packages/opentelemetry-instrumentation/src/platform/node/old/autoLoader.ts

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// This should be removed after plugins are gone
+
+import * as api from '@opentelemetry/api';
+import { NodePlugins, OldClassPlugin } from '../../../types_plugin_only';
+import { PluginLoader } from './PluginLoader';
+
+/** List of all default supported plugins */
+export const DEFAULT_INSTRUMENTATION_PLUGINS: NodePlugins = {
```

Comment:
do we still want those defaults there ? If later on we start to migrate from plugins to instrumentation (say only http/grpc), we'll need to update them here too. Could we move the default loading to the `@opentelemetry/node` package so all the default are handled there ?

Reply:
of course autoloader will be the only loader finally and we need to keep backward compatibility with loading old plugins automatically

Reply:
Yeah but could we set this `DEFAULT_INSTRUMENTATION_PLUGINS` inside `@opentelemetry/node` instead of here ? So when we want to use the new instrumentation by default, we just need to change the config there

Reply:
it will be removed from there